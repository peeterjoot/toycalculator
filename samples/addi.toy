INT32 x;
x = 5 + 3.14E0;
FLOAT64 f;
f = x;
PRINT f;
// PRINT now supports all the builtin types:
PRINT x;
