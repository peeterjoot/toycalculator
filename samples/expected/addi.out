8.000000
8
